components/messaging/ConversationCard.tsx

```diff
@@ -22,24 +23,28 @@ const ConversationCard: React.FC<ConversationCardProps> = ({
   joinedAt,
   itemName,
   imageSrc,
+  conversationId,
   clickHandler,
 }) => {
   return (
-    <button type='button' onClick={clickHandler}>
-      <div className='m-2 flex max-h-28 w-[400px] items-center gap-4 rounded-lg bg-gray-300 p-4 shadow-md hover:shadow-lg'>
+    <div tabIndex={0} aria-label='button' onClick={clickHandler}>
```

Comment:
Smart 🧠 

I figure it might be a bit unconventional but can't think of any better way to solve that button issue. Nice touch that you added the aria-label!